src/lib/_proxied-dot-io/get-static-generation-functions.ts

```diff
@@ -0,0 +1,25 @@
+import { getStaticGenerationFunctions as getStaticGenerationFunctionsBase } from '@hashicorp/react-docs-page/server'
+
+export const getStaticGenerationFunctions: typeof getStaticGenerationFunctionsBase = (
+  ...args
+) => {
+  const {
+    getStaticPaths: getStaticPathsBase,
+    getStaticProps,
+  } = getStaticGenerationFunctionsBase(...args)
+
+  return {
+    async getStaticPaths(ctx) {
+      const result = await getStaticPathsBase(ctx)
+
+      return {
+        ...result,
+        paths: result.paths.slice(
+          0,
+          Number.parseInt(process.env.IO_SITES_MAX_STATIC_PATHS, 10)
```

Comment:
What happens here if `IO_SITES_MAX_STATIC_PATHS` isn't defined? (eg since `config/development.json` is empty at the moment, that might happen if `HASHI_ENV` isn't set, I think?). At a glance it looks like we might end up with `NaN` as the second arg to `result.paths.slice`, but not sure 🤔